web/src/stream_settings_ui.ts

```diff
@@ -243,6 +244,13 @@ export function update_is_default_stream(): void {
 export function update_channel_folder(sub: StreamSubscription, folder_id: number | null): void {
     stream_data.update_channel_folder(sub, folder_id);
     stream_ui_updates.update_channel_folder_dropdown(sub);
+    stream_list.build_stream_list(false);
+    const $stream_li = stream_list.get_stream_li(sub.stream_id);
+    if ($stream_li) {
+        stream_list.scroll_stream_into_view($stream_li);
+    }
```

Comment:
Why are we scrolling the channel into view from here? This seems very odd, since it's an administrator action to edit channel folder membership.

Reply:
Oh, this is leftover from my misunderstanding that they were personal folders. Thanks for catching this!